pyiron_base/jobs/job/generic.py

```diff
@@ -137,12 +137,12 @@ def __init__(self, project, job_name):
             self.refresh_job_status()
         elif os.path.exists(self.project_hdf5.file_name):
             initial_status = read_hdf5(
-                self.project_hdf5.file_name, self.job_name + "/status"
+                self.project_hdf5.file_name, self.project_hdf5.h5_path + "/status"
             )
             self._status = JobStatus(initial_status=initial_status)
             if "job_id" in self.list_nodes():
                 self._job_id = read_hdf5(
-                    self.project_hdf5.file_name, self.job_name + "/job_id"
+                    self.project_hdf5.file_name, self.project_hdf5.h5_path + "/job_id"
```

Comment:
In most cases `self.project_hdf5.h5_path == / + self.job_name`  but when a job is stored in a GenericMaster this is no longer the case, so we have to use `self.project_hdf5.h5_path`.